### DISCUSSION TITLE
Why is the protocol called tea?

### DISCUSSION THREAD

**Discussion by huseyindurna** [teaxyz/white-paper#190] [General]
Why is the protocol called tea?

**Comment by mxcl** [comment]
It’s a play on my previous work Homebrew. At the time the consumption of beer and web2 went hand in hand. We feel web3 has matured from that base so now we drink tea.

**Reply by furkancelik** [reply]
It's very creative and very nice. Better than beer.. :)

**Comment by abelonuh** [comment]
I am inquisitive about this too.

**Comment by lyd168** [comment]
me too.

**Comment by mojinxun** [comment]
everyone like tea

**Comment by Samped** [comment]
smiles
